### PR TITLE
FISH-704 Increment version numbers

### PR DESCRIPTION
**Pull request by MarkWareham**

### CODE REVIEW

**Review by cubastanley — APPROVED** [review]

**Comment by cubastanley** [comment]
Jenkins test please

File: appserver/packager/appserver-base/src/main/docs/README.txt [hunk 1]
```diff
@@ -50,13 +50,13 @@ To stop Payara Server, just issue the following command:
 Open the following in your browser: https://docs.payara.fish/. It contains useful information such as the details about the Payara Project, links to the Payara Server Documentation, etc.
 
 
-Make sure to also check the Payara Server 5.2020.6 Release Notes as they contain important information: https://docs.payara.fish/docs/5.2020.6/release-notes/release-notes-2020.2.html
+Make sure to also check the Payara Server 5.2020.7 Release Notes as they contain important information: https://docs.payara.fish/docs/5.2020.7/release-notes/release-notes-2020.2.html
```
Comment (Pandrex247) [inline appserver/packager/appserver-base/src/main/docs/README.txt:53]:
```suggestion
Make sure to also check the Payara Server 5.2020.7 Release Notes as they contain important information: https://docs.payara.fish/docs/5.2020.7/release-notes/release-notes-2020-7.html
```
URL is wrong

**Review by Pandrex247 — CHANGES_REQUESTED** [review]
**Also** missing update to micro `glassfish-version.properties`

File: appserver/packager/appserver-base/src/main/docs/README.txt [hunk 2]
```diff
@@ -50,13 +50,13 @@ To stop Payara Server, just issue the following command:
...
 
 
 5. Documentation
 ================
 
-Payara Server 5.2020.6 Release Notes: https://docs.payara.fish/docs/5.2020.6/release-notes/release-notes-2020.2.html
+Payara Server 5.2020.7 Release Notes: https://docs.payara.fish/docs/5.2020.7/release-notes/release-notes-2020.2.html
```
Comment (Pandrex247) [inline appserver/packager/appserver-base/src/main/docs/README.txt:59]:
```suggestion
Payara Server 5.2020.7 Release Notes: https://docs.payara.fish/docs/5.2020.7/release-notes/release-notes-2020-7.html
```

**Review by Pandrex247 — CHANGES_REQUESTED** [review]
I can't edit `glassfish-version.properties` from here since you haven't made a change near it. Don't forget it please, it's wrong.

File: appserver/extras/payara-micro/payara-micro-boot/src/main/resources/MICRO-INF/domain/branding/glassfish-version.properties [hunk 3]
```diff
@@ -3,7 +3,7 @@ brief_product_name=@@@PRODUCT@@@
 abbrev_product_name=@@@PRODUCT@@@
 major_version=5
 minor_version=2020
-update_version=4-SNAPSHOT
+update_version=7-SNAPSHOT
```
Comment (cubastanley) [inline appserver/extras/payara-micro/payara-micro-boot/src/main/resources/MICRO-INF/domain/branding/glassfish-version.properties:6]:
```suggestion
update_version=7
```

**Review by Pandrex247 — APPROVED** [review]

**Review by cubastanley — CHANGES_REQUESTED** [review]
Scripts don't account for a snapshot in here and, although I may be overlooking something, I don't see that it needs to be snapshot in this file

**Review by cubastanley — APPROVED** [review]

**Comment by MarkWareham** [comment]
jenkins test please

**Comment by MarkWareham** [comment]
Overrode Jenkins check as it is a false positive

**Pull request merged**